mmdet/models/utils/ckpt_convert.py

```diff
@@ -0,0 +1,73 @@
+from collections import OrderedDict
```

Comment:
This script is not general enough. Do we really need this script?

Reply:
Refer to https://github.com/open-mmlab/mmsegmentation/pull/791.
Move converter to tools and add usage to README.